Support `contrib/scripts/kind.sh` on macOS

The script only runs on Linux at the moment, failing on macOS with the error - 

```
$ make kind WORKERS=0
./contrib/scripts/kind.sh
readlink: illegal option -- e
usage: readlink [-n] [file ...]
usage: dirname path
./contrib/scripts/kind.sh: line 22: realpath: command not found
make: *** [kind] Error 127
```

--- a/contrib/scripts/kind.sh
+++ b/contrib/scripts/kind.sh
@@ -19,7 +19,7 @@ cluster_name="${3:-${CLUSTER_NAME:=${default_cluster_name}}}"
 image="${4:-${IMAGE:=${default_image}}}"
 kubeproxy_mode="${5:-${KUBEPROXY_MODE:=${default_kubeproxy_mode}}}"
 ipfamily="${6:-${IPFAMILY:=${default_ipfamily}}}"
-CILIUM_ROOT="$(realpath $(dirname $(readlink -ne $BASH_SOURCE))/../..)"
+CILIUM_ROOT="$(git rev-parse --show-toplevel)"
 
 usage() {
   echo "Usage: ${PROG} [control-plane node count] [worker node count] [cluster-name] [node image] [kube-proxy mode] [ip-family]"

